scripts/checkComponentAppProp.js

```diff
@@ -14,8 +14,12 @@ function checkComponentKey(component, nameslug) {
     console.error(`[!] ${key} - missing app prop`);
```

Comment:
This function should be renamed right? It's not `checkingComponentKey` and it could be simplified?

```js
function checkComponentHasAppProp(component, appNameSlug) {
  const matching = Object.values(component.props).filter((prop) => prop.type === "app" && prop.app === appNameSlug);
  if (!matching.length) {
    console.error(`[!] ${component.key} - missing app prop for ${appNameslug}`)
    err = true
  }
}
```